resources/function-controller/values.yaml

```diff
@@ -31,12 +36,15 @@ resources:
     cpu: 500m
     memory: 256Mi
 
+metrics:
+  enabled: false
+  service:
+    port:
+      name: https
+      external: 8443
+  serviceMonitor:
+    create: false
```

Comment:
Please also add field `scrapeInterval: 30s` and use it in service-monitor resource.